tests/test_all_rules.py

```diff
@@ -394,3 +395,46 @@ def test_ecs_and_beats_opt_in_not_latest_only(self):
                 error_msg = f'{error_prefix} it is unnecessary to define the current latest ecs version if only ' \
                             f'one version is specified: {latest_ecs}'
                 self.assertNotIn(latest_ecs, ecs_versions, error_msg)
+
+
+class TestTuleTiming(unittest.TestCase):
```

Comment:
@rw-access I added tests for this as well as to check that the longer lookback is defined as needed for consistency (especially since EQL sequence rules will not use the override).

There were ~50 rules which did not define the extended lookback that were targeting the endpoint index, so I updated those rules as well

Reply:
Do we need the longer look back if we've already set the timestamp override? I wouldn't want to be too conservative since it comes with a performance cost

Reply:
Well this definitely discovered an inconsistency that needing fixing

If the override gives back confidence to the default `now-5m` then I can update the test to only check eql-sequence rules to ensure the longer lookback. For all the rest, they should be consistent, whether it is the default or 4 min extended lookback.

What do you think @spong - is it worth converting the lookback back to the default `now-5m` for all rules which define the override or is there any reason to leave the extended lookback

Reply:
In our conversations with @MikePaquette and the endpoint folks, the outcome was to add the timestamp override and to not touch the lookbacks. 

Was able to find this quote: 

> The timestamp override should eliminate the effects of ingestion pipeline delay causing missed alerts, which means for those rules, the additional look-back time would not offer any further benefit. I was voting for not changing the lookback time if we did the timestamp override.

Reply:
Ok, that sounds right. I think we should def be consistent with the rules that have the override defined, whether it is the default or 9m (whereas right now it wasn't fully enforced, hence the test). 

I guess _if_ it ever needed to fallback to `@timestamp`, that would be the only scenario where the extended lookback makes sense. Actually, would it make sense to add that to the override logic in Kibana? That if the override is defined, defer to the default lookback of `now-5m`?

I think I will leave the 9m lookback for all the rules with the override defined in this PR and if it is decided that stripping it out is worth the performance gain, we can do that in a future PR (for 7.12) or future release